src/index.js

```diff
@@ -1,15 +1,51 @@
-import _ from 'lodash';
 import './style.css';
-import Icon from './icon.png';
 
- function component() {
-   const element = document.createElement('div');
+const listContents = document.querySelector('.listContents');
 
-   // Lodash, now imported by this script
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+const data = [
+  {
+    descript: 'ghislain Developer',
```

Comment:
[OPTIONAL]

- I would recommend you make your naming convention descriptive and easy to understand for the easy maintainability of your website 👍🏼 . Using `description` instead of `descript` is better